### DISCUSSION TITLE
Validation for nested array is not working

### DISCUSSION THREAD

**Discussion by LaaHudra** [livewire/livewire#7817] [7. Bugs (archived)]
### Livewire version

v3.4.1

### Laravel version

v10.42.0

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

Create a rules method with validation for nested elements.

### Repository or Wirebox URL

https://wirebox.app/b/4mpz0

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by perisicnikola37** [comment]
Got: Sorry, this Wirebox is private.

Please provide me URL to a GitHub repository so I can help you :)

**Reply by joshhanley** [reply]
Or just make the wirebox public 🙂

**Reply by perisicnikola37** [reply]
> Or just make the wirebox public 🙂

I prefer repository because I can immediately send a PR with modifications :) But wirebox is also okay
